### ISSUE TITLE
sprite layering in 3d sprig is wrong

### ISSUE THREAD

**Issue by Omay238** [hackclub/sprig#425]

**Comment by RoloTheDevBunny** [comment]
The z index, or layout, is controlled by the array where you define all your variables and your bit maps. Top is on top, bottom is on the bottom. Hope this helps!

**Comment by Omay238** [comment]
I know, but if you play it in the default editor, the layering is correct. There is an issue with the rendering of the game on the 3D sprig.

**Comment by RoloTheDevBunny** [comment]
If it is layered correctly on a 2D device what does 3D have to do with this? Could you send a short video of your problem?

**Comment by Omay238** [comment]
[2d.webm](https://user-images.githubusercontent.com/70179858/193862806-8bfa8c78-f8d5-46a7-81d3-57ae51d02299.webm)
[3d.webm](https://user-images.githubusercontent.com/70179858/193862866-df9cc6b0-20f2-4d86-ac3d-6fe90d79f0e9.webm)

**Comment by grymmy** [comment]
@Omay238 this is absolutely ancient, but can you please fill me in on exactly what you mean by the 3d sprig?  Do you mean the PC simulator version?  Do you happen to know if this still occurs?  Apologies for the incredibly slow response...

**Comment by RoloTheDevBunny** [comment]
I believe he is referring to Z-Index. That was the closest thing I could think of relating to 3D layering in a 2d plane. I explained up above a solution for this.

**Comment by Omay238** [comment]
On the 3D Sprig on the main page, the sprite layering is broken. I'll check if it's still an issue

**Comment by Omay238** [comment]
Yeah, it's still an issue. If you press the d key on the 3D sprig, than the a key, then the real player (black one) will go behind the gray one, even though in the code, it should be going on top.
